### PR TITLE
BaSP-TG-19: Create Share Component Folder

### PR DESCRIPTION
**Pull request by JulianFloresDev**
- Create folder to Share components.
- Create fisrt Share component -> Table Component
- Create second Share component -> Table Header Component with filter option

### CODE REVIEW

File: src/Components/Layout/index.js [hunk 1]
```diff
@@ -17,12 +17,12 @@ function Layout() {
...
-          <Route path={'/employees'} component={Employees} />
-          <Route path={'/projects'} component={Projects} />
-          <Route path={'/time-sheets'} component={TimeSheets} />
-          <Route path={'/tasks'} component={Tasks} />
+          <Route exact path={'/admins'} component={Admins} />
+          <Route exact path={'/super-admins'} component={SuperAdmins} />
+          <Route exact path={'/employees'} component={Employees} />
+          <Route exact path={'/projects'} component={Projects} />
+          <Route exact path={'/time-sheets'} component={TimeSheets} />
+          <Route exact path={'/tasks'} component={Tasks} />
```
Comment (Alfondalix) [inline src/Components/Layout/index.js:25]: Good catch

File: src/Components/Share/Table/index.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import styles from './table.module.css';
+import { useHistory } from 'react-router-dom';
+// import Modal from '../Modal';
+const Table = ({ headers, data, setList }) => {
+  const history = useHistory();
+  const URLPath = history.location.pathname.split('/');
+  const entitie = URLPath[1];
+  const deleteRow = () => {
+    setList([]);
+    console.log('Open Modal Component');
```
Comment (Alfondalix) [inline src/Components/Share/Table/index.js:10]: We should delete this line

File: src/Components/Share/Table/index.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  };
+  const openListModal = (list) => {
+    console.log('Open Modal Component with employee List:', list);
```
Comment (Alfondalix) [inline src/Components/Share/Table/index.js:13]: Same as above

File: src/Components/Share/Table/index.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                                  </>
+                                );
+                              }
+                              return (
+                                <li key={item._id} className={styles.optionInvalid}>
+                                  Employee Not Found!
+                                </li>
+                              );
+                            })}
+                          </ul> */}
```
Comment (Alfondalix) [inline src/Components/Share/Table/index.js:83]: What about this commented lines? If we're not going to use them we should delete them

**Review by Alfondalix — CHANGES_REQUESTED** [review]

File: src/Components/Share/Table/index.js (hunk 4 above)
Comment (JulianFloresDev) [reply src/Components/Share/Table/index.js:83]: That was from old Wayne to send employee list from teamMmbers. After meet we decide remove this. It should be remove

File: src/Components/Share/Table/index.js (hunk 3 above)
Comment (JulianFloresDev) [reply src/Components/Share/Table/index.js:13]: It still in progress, but should be change for a real function

File: src/Components/Share/Table/index.js (hunk 2 above)
Comment (JulianFloresDev) [reply src/Components/Share/Table/index.js:10]: Still in progress, same as 👇

**Review by kolodani — APPROVED** [review]
Well Done!!!

**Review by DinoStampella — APPROVED** [review]

**Review by LuisQueff — APPROVED** [review]

**Review by Alfondalix — APPROVED** [review]

**Pull request merged**